meta-facebook/minerva-ag/src/platform/plat_event.c

```diff
@@ -184,6 +184,6 @@ void init_cpld_polling(void)
 	cpld_polling_tid = k_thread_create(
 		&cpld_polling_thread, cpld_polling_stack, K_THREAD_STACK_SIZEOF(cpld_polling_stack),
 		poll_cpld_registers, NULL, NULL, NULL, CONFIG_MAIN_THREAD_PRIORITY, 0,
-		K_MSEC(3000)); //sleep for 3 seconds to prevent dc status changing when reboot BIC
+		K_MSEC(2000)); //sleep for 3 seconds to prevent dc status changing when reboot BIC
```

Comment:
Can you please change the comment as well to reflect 2 instead of 3?

Reply:
Thank you for the feedback. I have updated the comment to clarify the behavior. The total delay before accessing CPLD is 3 seconds, not 2. Here's the breakdown:

1. 2-second delay comes from K_MSEC(2000) in k_thread_create.
2. 1-second delay comes from CPLD_POLLING_INTERVAL_MS at the start of the thread handler loop.